.yamato/com.unity.ml-agents-test.yml

```diff
@@ -1,152 +1,141 @@
 test_editors:
-  - version: 2018.4
-    # 2018.4 doesn't support code-coverage
-    enableCodeCoverage: !!bool false
-    # We want some scene tests to run in the DevProject, but packages there only support 2019+
-    testProject: Project
-    enableNoDefaultPackages: !!bool false
-  - version: 2019.4
-    enableCodeCoverage: !!bool true
-    testProject: DevProject
-    enableNoDefaultPackages: !!bool true
-  - version: 2020.2
-    enableCodeCoverage: !!bool true
-    testProject: DevProject
-    enableNoDefaultPackages: !!bool true
+    - version: 2018.4
+        # We want some scene tests to run in the DevProject, but packages there only support 2019+
+      testProject: Project
+      enableNoDefaultPackages: !!bool false
+    - version: 2019.4
+      testProject: DevProject
+      enableNoDefaultPackages: !!bool true
+    - version: 2020.2
+      testProject: DevProject
+      enableNoDefaultPackages: !!bool true
 
 trunk_editor:
-  - version: trunk
-    # Workaround for MLA-1596 - need to make sure we load the right results.
-    enableCodeCoverage: !!bool false
-    testProject: DevProject
+    - version: trunk
+        # Workaround for MLA-1596 - need to make sure we load the right results.
+      enableCodeCoverage: !!bool false
+      testProject: DevProject
 
 test_platforms:
-  - name: win
-    type: Unity::VM
-    image: package-ci/win10:stable
-    flavor: b1.large
-  - name: mac
-    type: Unity::VM::osx
-    image: package-ci/mac:stable
-    flavor: b1.small
-  - name: linux
-    type: Unity::VM
-    image: package-ci/ubuntu:stable
-    flavor: b1.medium
+    - name: win
+      type: Unity::VM
+      image: package-ci/win10:stable
+      flavor: b1.large
+    - name: mac
+      type: Unity::VM::osx
+      image: package-ci/mac:stable
+      flavor: b1.small
+    - name: linux
+      type: Unity::VM
+      image: package-ci/ubuntu:stable
+      flavor: b1.medium
 
 packages:
-  - name: com.unity.ml-agents
-    assembly: Unity.ML-Agents
-    minCoveragePct: 72
-  - name: com.unity.ml-agents.extensions
-    assembly: Unity.ML-Agents.Extensions*
-    minCoveragePct: 75
+    - name: com.unity.ml-agents
+      assembly: Unity.ML-Agents
+      minCoveragePct: 72
+    - name: com.unity.ml-agents.extensions
+      assembly: Unity.ML-Agents.Extensions*
+      minCoveragePct: 75
 ---
 
 all_package_tests:
-  name: Run All Combinations of Editors/Platforms Tests
-  dependencies:
+    name: Run All Combinations of Editors/Platforms Tests
+    dependencies:
+      - .yamato/com.unity.ml-agents-coverage.yml#test_coverage_com.unity.ml-agents_linux_2019.4
```

Comment:
This is hardcoded to `2019.4` but we're using `test_coverage_{{ package.name }}_{{ platform.name }}_{{ editor.version }}` with
```
test_editors:
    - version: 2020.2
      testProject: DevProject
```

so I think this will never run

Reply:
d'oh